clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -368,7 +368,8 @@ Changes in existing checks
 - Improved :doc:`readability-implicit-bool-conversion
   <clang-tidy/checks/readability/implicit-bool-conversion>` check to provide
   valid fix suggestions for ``static_cast`` without a preceding space and
-  fixed problem with duplicate parentheses in double implicit casts.
+  fixed problem with duplicate parentheses in double implicit casts. Corrected
+  the fix suggestions for C23 by using C-style casts instead of ``static_cast``.
```

Comment:
C-style casts are inserted for all C standards not only C23, aren't they?

Reply:
The checker should only affect C23 (+later) of all C standards since the checker has
```
  bool isLanguageVersionSupported(const LangOptions &LangOpts) const override {
    return LangOpts.Bool;
  }
```
C23 is the first C standard with the bool keyword, instead of provided via macros, and my understanding is that's what LangOpts.Bool indicates.

Reply:
That makes sense, even though it's not super obvious at first.